src/entity_extraction/training/spacy_ner/run_evaluation.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh 
+
+# if running with conda envs, comment out if not
+conda activate fossil_lit
+
+echo python.__version__ = $(python -c 'import sys; print(sys.version)')
+# ensure we're in the MetaExtractor root directory
+echo "Current working directory: $(pwd)"
+
+# set the location of the labelled data, ideally this is run from root of repo
+export DATA_DIR="$(pwd)/data/entity-extraction/processed/2023-05-31_label-export_39-articles/val"
+export MODEL_PATH="$(pwd)/models/ner/transformer-v3"
+export OUTPUT_DIR="$(pwd)/results/ner/test-results/"
```

Comment:
nitpick: if we can standardize on results folders being called `evaluation-results` it'll make summarization nice!
```suggestion
export OUTPUT_DIR="$(pwd)/results/ner/evaluation-results/"
```